src/bottomSection/pageNumber.tsx

```diff
@@ -0,0 +1,34 @@
+import { Component } from 'react';
+import { Counter } from './counter';
+
+export class PageWrapper extends Component {
+  render() {
+    let numberOfPages: number = Math.ceil(Number(localStorage.getItem('pagesNumber')) / 10);
```

Comment:
ESLint: 'numberOfPages' , wrapperPages, currentCount is never reassigned. Use 'const' instead.(prefer-const)